contrib/storage-hive/core/src/main/java/org/apache/drill/exec/store/hive/schema/DrillHiveTable.java

```diff
@@ -179,4 +179,9 @@ private void throwUnsupportedHiveDataTypeError(String hiveType) {
 
     throw new RuntimeException(errMsg.toString());
   }
+
+  @Override
+  public boolean providesDeferredSchema() {
```

Comment:
I believe that there other places (the select * replacement) which depend on the same type of property. We should use a single method to determine this. @jinfengni , can you point Sudheesh in the right direction?